packages/interactive_media_ads/lib/src/ads_request.dart

```diff
@@ -11,10 +11,53 @@ class AdsRequest {
   AdsRequest({
     required String adTagUrl,
     ContentProgressProvider? contentProgressProvider,
+    bool? adWillAutoPlay,
+    bool? adWillPlayMuted,
+    bool? continuousPlayback,
+    double? contentDuration,
+    List<String>? contentKeywords,
+    String? contentTitle,
+    double? liveStreamPrefetchSeconds,
+    double? vastLoadTimeout,
```

Comment:
This should also be a `Duration?` if possible. Especially since the units are different.